freeipa-perftest.spec

```diff
@@ -0,0 +1,84 @@
+Name:           freeipa-perftest
+Version:        0.1
+Release:        1%{?dist}
+Summary:        A set of IPA performance tools
+
+License:        GPLv3
+URL:            https://github.com/freeipa/freeipa-perftest.git
+Source0:        https://github.com/freeipa/freeipa-perftest/archive/%{version}.tar.gz
+
+BuildRequires:  pam-devel
+BuildRequires:  krb5-devel
+BuildRequires:  popt-devel
+BuildRequires:  glibc-devel
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  gcc
```

Comment:
We need to add `python3-devel` to BuildRequires.

Reply:
Added